test/IRGen/autolink-coff.swift

```diff
@@ -15,3 +24,10 @@ import module
 // CHECK-GNU-ASM: .section .drectve
 // CHECK-GNU-ASM: .ascii " -lmodule"
 
+// CHECK-MSVC-IR: !{{[0-9]+}} = !{i32 {{[0-9]+}}, !"Linker Options", [[NODE:![0-9]+]]}
+// CHECK-MSVC-IR: [[NODE]] = !{[[LIST:![0-9]+]]}
+// CHECK-MSVC-IR: [[LIST]] = !{!"/DEFAULTLIB:module.lib"}
+
+// CHECK-MSVC-ASM: .section .drectve
+// CHECK-MSVC-ASM: .ascii " /DEFAULTLIB:module.lib"
+
```

Comment:
I can even upload them!  I already have them ready to go, just waiting on the PS4 support to be merged :-).  I can remove this handling and make it part of the PS4 target support if you like.